GTG/backends/bugzilla/bug.py

```diff
@@ -76,7 +64,17 @@ class RedHatBug(Bug):
 }
 
 
-class BugFactory(object):
-    @staticmethod
-    def create(serviceDomain, bug):
-        return bugs[serviceDomain](bug)
+def create_bug(service_domain, bug):
```

Comment:
What's the purpose of this whole module? To me it's enough to just check if  the service_domain is in the list of whitelisted domains, something like

```python

DOMAINS = set([
  'bugzilla.gnome.org',  'bugs.freedesktop.org', 'bugzilla.mozilla.org', 
  'bugzilla.samba.org', 'bugs.gentoo.org', 'bugzilla.redhat.com',
])
if service_domain not in DOMAINS:
  return None
```